pages/politicians/[slug].tsx

```diff
@@ -130,11 +129,9 @@ const PoliticianPage: NextPage<{ mobileNavTitle?: string }> = ({
       [politicianStyles.wide as string]: tags.length === 0,
     });
 
-    if (!termStart && !termEnd && !yearsInPublicOffice && !age) return null;
```

Comment:
We need this line otherwise this section will say "Basic Info" even when there is no data